docs/overview.md

```diff
@@ -0,0 +1,43 @@
+# Overview
+
+The Foreman project runs a number of different servers for testing, packaging, and continuous integration. Most of these servers are available as Jenkins build slaves for http://ci.theforeman.org.
+
+## Access
+
+Access to Foreman project infrastructure is available for those who wish to assist in building packages, testing, and building Jenkins jobs. Fork http://github.com/theforeman/foreman-infra and add an ssh_user resource to the users module (see puppet/modules/users/manifests/init.pp) and your key into the files directory. Submit a pull request to the infrastructure project and then talk to ? in #theforeman-dev on irc.freenode.net. One of them can merge your change and update the puppetmaster.
+
+## Landscape
+
+
+| Host | Provider | Function |
+|---|---|---|
+| ci.theforeman.org | OSUOSL | Jenkins |
+|   |   |   |
+|   |   |   |
+
+## Infrastructure providers
+
+A list of the hosting we have, who provides it, and what capacity it has
+
+* Rackspace
+** $2k per month
+** Hosts web02, the Elastic logging instance, Jenkins, and some permanent slaves
+** Contact support from our account as needed
+* Scaleway
+** $25 per month (not a lot!), but they're cheap so we do OK. Hosts Discourse, Redmine, and some ARM build slaves
+** Support usually helpful, Edouard Bonlieu & Yann Léger were initial contacts
+* Fastly
+** $1k/month CDN
+** Elaine Greenberg was initial contact
+* OSUOSL
+** Free
+** Hosts test machines, slaves, and stats.tf.o
+** Contact Lance Alberston if more capacity is needed
+* NETWAYS
+** Free
+** Contributing a single VM for the Builders/RH hostgroup
```

Comment:
We actually have an account in their openstack instance with capacity for more than one. I just had issues installing and haven't gotten back to their support.